deployment/hasura/metadata/actions.graphql

```diff
@@ -6,6 +6,13 @@ type Mutation {
   ): AddExternalDatasetResponse
 }
 
+type Mutation {
+  uploadDictionary(dictionary: String!): CommandDictionaryResponse
+}
+type CommandDictionaryResponse {
```

Comment:
Would you mind moving `CommandDictionaryResponse` to further down in this file?

Looks like this file has this rough ordering:
- Mutations
- Queries
- Types needed for mutations & queries (`*Response` types)
- Scalars

Reply:
@pcrosemurgy  I can fix this on the Upload Command Dictionary PR so we can resolve this.

Reply:
Thanks, that'd be nice